### PR TITLE
Updated contribution documentation.

### PR DESCRIPTION
**Pull request by evildmp**

### CODE REVIEW

File: docs/contributing/development-community.rst [hunk 1]
```diff
@@ -1,15 +1,37 @@
-#########################
-Development of django CMS
-#########################
+..  _development-community:
 
-django CMS is developed by a community of developers from across the world,
-with a wide range and levels of skills and expertise. Every contribution,
-however small, is valued.
+##################################
+django CMS's development community
```
Comment (FinalAngel) [inline docs/contributing/development-community.rst]: why not "Contributing to django CMS" ?

Comment (evildmp) [inline docs/contributing/development-community.rst]: That page isn't really about contributing; it's about all the different groups and people who make up the community.

Comment (FinalAngel) [inline docs/contributing/development-community.rst]: 👍

File: docs/contributing/development-community.rst [hunk 2]
```diff
@@ -1,15 +1,37 @@
...
+You can also follow:
+
+* the `Travis Continuous Integration build reports <https://travis-ci.org/divio/django-cms>`_
+* the `@djangocms`_ Twitter account for general announcements
+
+You don't need to be an expert developer to make a valuable contribution - all
+you need is a little knowledge of the system, and a willingness to follow the
+contribution guidelines.
+
+Remember that contributions to the documentation are highly prized, and key to
```
Comment (FinalAngel) [inline docs/contributing/development-community.rst]: I'd write "highly appreciated"

File: docs/contributing/development-policies.rst [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+
+Reviews are welcomed by all members of the community. You don't need to be a core developer, or even an experienced
+programmer, to contribute usefully to code review. Even noting that you don't understand something in a pull request
+is valuable feedback and will be taken seriously.
+
+
+Formal approval
+===============
+
+Formal approval means "OK to merge" comments, following review, from at least two different members of the core team
```
Comment (FinalAngel) [inline docs/contributing/development-policies.rst]: "OK to merge, LGTM, +1 etc."?

File: docs/contributing/development-policies.rst [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+******
+
+All patches should be made as pull requests to `the GitHub repository <https://github.com/divio/django-cms>`_. Patches
+should never be pushed directly.
+
+**Nothing** may enter the code-base, *including the documentation*, without proper review and formal approval from the
+core team.
+
+Reviews are welcomed by all members of the community. You don't need to be a core developer, or even an experienced
+programmer, to contribute usefully to code review. Even noting that you don't understand something in a pull request
```
Comment (FinalAngel) [inline docs/contributing/development-policies.rst]: ``to contribute usefully to code review`` sounds a bit weird to me.

File: docs/contributing/development-policies.rst [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+
+        or in any other public forum until we have had a chance to deal with it.
+
+
+******
+Review
+******
+
+All patches should be made as pull requests to `the GitHub repository <https://github.com/divio/django-cms>`_. Patches
+should never be pushed directly.
```
Comment (FinalAngel) [inline docs/contributing/development-policies.rst]: eventually do already a reference to the branching system here.

File: docs/contributing/development-policies.rst [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+is valuable feedback and will be taken seriously.
+
+
+Formal approval
+===============
+
+Formal approval means "OK to merge" comments, following review, from at least two different members of the core team
+who have expertise in the relevant areas, and excluding the author of the pull request.
+
+The exceptions to this are frontend code and documentation, where one "OK to merge" comment will suffice, at least
```
Comment (FinalAngel) [inline docs/contributing/development-policies.rst]: ``where one single "OK to merge" comment``

File: docs/contributing/development-policies.rst [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+until the team has more expert developers in those areas.
```
Comment (FinalAngel) [inline docs/contributing/development-policies.rst]: reference to "how you can join us to help"?

File: docs/contributing/development-policies.rst [hunk 8]
```diff
@@ -0,0 +1,219 @@
...
+**********************************************
+Proposal and discussion of significant changes
+**********************************************
+
+New features and backward-incompatible changes should be proposed using the `django CMS developers email list
+<http://groups.google.com/group/django-cms-developers>`_. Discussion should take place there before any pull requests
+are made.
+
+This is in the interests of openness and transparency, and to give the community a chance to participate in and
+understand the decisions taken by the project.
```
Comment (FinalAngel) [inline docs/contributing/development-policies.rst]: Questions asked on GitHub will be closed and redirected to these resources?

File: docs/contributing/development-policies.rst [hunk 9]
```diff
@@ -0,0 +1,219 @@
...
+****************
+
+The `roadmap <https://github.com/divio/django-cms/wiki/Roadmap>`_ can be found on our GitHub wiki
+page.
+
+We are planning releases according to **key principles and aims**. Issues within milestones are
+therefore subject to change.
+
+The `django CMS developers email list <http://groups.google.com/group/django-cms-developers>`_ serves as gathering
+point for developers. We submit ideas and proposals prior to the roadmap goals.
```
Comment (FinalAngel) [inline docs/contributing/development-policies.rst]: @evildmp do we somewhere also describe that the roadmap is set by the technical board according to feedback in the developers group and the core developers? Eventually a reference here or additional content.

File: docs/contributing/development-policies.rst [hunk 10]
```diff
@@ -0,0 +1,219 @@
...
+
+..  versionchanged:: 3.3
+
+    Previously, we maintained a ``master`` branch (now deleted), and a set of ``support`` branches (now pruned, and
+    renamed ``release``).
+
+We maintain a number of branches on `our GitHub repository <https://github.com/divio/django-cms>`_.
+
+the latest (highest-numbered) ``release/x.y.z``
+    This is the branch that will become the next release on PyPI.
```
Comment (FinalAngel) [inline docs/contributing/development-policies.rst]: the next "minor" release on PyPi?

File: docs/contributing/development-policies.rst [hunk 11]
```diff
@@ -0,0 +1,219 @@
...
+
+    **Fixes and backwards-compatible improvements** (i.e. most pull requests) will be made against
+    this branch.
+
+``develop``
+    This is the branch that will become the next release that increments the ``x`` or ``y`` of the latest
```
Comment (FinalAngel) [inline docs/contributing/development-policies.rst]: the next "major2 release

File: docs/contributing/development-policies.rst [hunk 12]
```diff
@@ -0,0 +1,219 @@
...
+    This branch is for **new features and backwards-incompatible changes**. By their nature, these will require more
+    substantial team co-ordination.
+
+Older ``release/x.y.z`` branches
+     These represent the final point of development (the highest ``y`` of older versions). Releases in the full set of
+     older versions have been tagged (use Git Tags to retrieve them).
+
+     These branches will only rarely be patched, with security fixes representing the main reason for a patch.
+
+Commits in ``release/x.y.z`` will be merged forward into ``develop`` periodically by the core developers.
```
Comment (FinalAngel) [inline docs/contributing/development-policies.rst]: Pull requests need to be forward ported not just a general "let's move everything now" as discussed. We might want to clarify this here.

**Comment by FinalAngel** [comment]
Great work @evildmp I think this will clarify several things :)

**Pull request merged**